SQL/0000-00-01-Permission.sql

```diff
@@ -102,6 +102,8 @@ INSERT INTO `permissions` VALUES
     (53,'publication_view', 'Publication - Access to module', 2),
     (54,'publication_propose', 'Publication - Propose a project', 2),
     (55,'publication_approve', 'Publication - Approve or reject proposed publication projects', 2);
```

Comment:
It should be a Comma at the end of this line, not a Semicolon, which is the reason Travis failed.

Reply:
thank you @kongtiaowang! Should be fixed in the last commit.

Reply:
Shouldn't that result in a syntax error while sourcing the SQL, and not MySQL freezing?

Reply:
yeah, I agree, this is weird... But this is passing Travis now, YEAH!!!!